### PR TITLE
#63

### PR DESCRIPTION
**Pull request by Esmaeeilenani**
register and login for client user

### CODE REVIEW

File: carRental/src/main/java/com/amigoscode/chohort2/carRental/spcefication/Util.java [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+
+    static <T> Predicate equal(Root<T> root, CriteriaBuilder cb,
+                               SingularAttribute<? super T, String> attributeName, String input) {
+        if (StringUtils.isNotBlank(input)) {
+            return cb.equal(root.get(attributeName), input.trim());
+        }
+        return null;
+    }
+
```
Comment (DmitriKonnovNN) [inline carRental/src/main/java/com/amigoscode/chohort2/carRental/spcefication/Util.java:29]: I reckon, I'm gonna make use of the methods in Search implementation, aren't I ?!

**Review by DmitriKonnovNN — APPROVED** [review]
Looks great. I'mma take my time to get hands on those Utils classes

File: carRental/src/main/java/com/amigoscode/chohort2/carRental/spcefication/Util.java (hunk 1 above)
Comment (Esmaeeilenani) [reply carRental/src/main/java/com/amigoscode/chohort2/carRental/spcefication/Util.java:29]: yes you will use more than one in fact

Comment (DmitriKonnovNN) [reply carRental/src/main/java/com/amigoscode/chohort2/carRental/spcefication/Util.java:29]: Yeah that piece of code is the scariest one. :D

**Pull request merged**